Tests/ParseSwiftTests/ParseObjectTests.swift

```diff
@@ -47,6 +77,18 @@ class ParseObjectCommandTests: XCTestCase { // swiftlint:disable:this type_body_
         try? ParseStorage.shared.deleteAll()
     }
 
+    func testObjectCreationViaStruct() {
+        XCTAssertEqual(GameScore.className, "GameScore")
+        XCTAssertEqual(GameScore(score: 10).className, "GameScore")
+    }
+
+    func testObjectCreationViaClass() {
```

Comment:
Is this necessary? This feels more like it's testing Swift than our code.

Reply:
I added these tests because I ran into an issue when I was using the framework within an app.

You are right, the test isn't needed.

We should add a note in the documentation that the `ParseObject` protocol should only be used with `structs` or `final classes` as in the `class` scenario if you subclass, you will only get the base class name as the name of the ParseObject table. This differs from the objc framework of course, so users transitioning from that one may not realize this